rules_building_block/discovery_generic_process_discovery.toml

```diff
@@ -29,13 +29,12 @@ type = "eql"
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
   (
-   process.name == "reg.exe" and process.args : "query" or
-   (process.name: ("powershell.exe", "pwsh.exe", "powershell_ise.exe") and
-    (process.args: ("*Get-ChildItem*", "*Get-Item*", "*Get-ItemProperty*") and
-     process.args : (
-      "*HKLM*", "*HKCU*", "*HKEY_LOCAL_MACHINE*", "*HKEY_CURRENT_USER*", "Registry::"
-   )))
-  )
+    process.name :("PsList.exe", "qprocess.exe") or 
+   (process.name : "powershell.exe" and process.args : ("*get-process*", "*Win32_Process*")) or 
+   (process.name : "wmic.exe" and process.args : ("process", "*Win32_Process*")) or
+   (process.name : "tasklist.exe" and not process.args : ("pid eq*")) or
```

Comment:
Curious about this one, if `pid eq*` is entered in the commandline, wouldn't `pid` end up being 1 `process.arg` and `eq*` being another `process.arg`? If that's the case, would it make sense to check `process.command_line` here?

Reply:
The pattern that I'm excluding here puts this between quotes so that you will have something like this in the args: 
![image](https://github.com/elastic/detection-rules/assets/26856693/f98ca6bd-4bc3-4bbf-bf86-1a697838a67a)